domain/src/unit-test/java/uk/gov/moj/sdt/domain/visitor/AbstractDomainObjectVisitorTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.moj.sdt.domain.visitor;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import uk.gov.moj.sdt.domain.api.*;
+import uk.gov.moj.sdt.utils.visitor.Tree;
+
+import static org.junit.Assert.assertThrows;
+
+class AbstractDomainObjectVisitorTest {
+
+    public AbstractDomainObjectVisitor abstractDomainObjectVisitor;
+
+    private static final String EXPECTED_MESSAGE =
+        "Missing visitor implementation: this method should never be called.";
+
+    @Mock
+    Tree mockTree;
```

Comment:
I think this will need to be initialised by either adding ```MockitoAnnotations.openMocks()``` to setUp() or annotating the class with ```@ExtendWith(MockitoExtension.class)```